### DISCUSSION TITLE
串口为什么检测不到STLINK?

### DISCUSSION THREAD

**Discussion by 477996850** [wh201906/SerialTest#86] [Q&A]
![63fa7e731565b7979d09cabbd86e05e](https://github.com/user-attachments/assets/53233323-8bb1-420a-a43d-4b66371b0f48)
![727f7780e28ec2500b756e8755a5242](https://github.com/user-attachments/assets/54d8f091-9bfd-4662-b0e7-c7a691a48a3a)

**Comment by wh201906** [comment]
你的stlink不带串口功能，因此检测不到
